dactyl-config.yml

```
@@ -4248,6 +4248,14 @@ pages:
         targets:
             - en
 
+    -   name: Developer Funding
+        html: developer-funding.html
+        parent: contribute.html
+        template: page-funding.html.jinja
+        sidebar: disabled
+        targets:
+            - en
+
```

Add the untranslated page to the Japanese target just so it's not a missing link there.

```suggestion
            - en
            - ja

```